docs/best-practices.md

```diff
@@ -111,7 +111,7 @@ export default ComponentName
   ```tsx
   // Example of styling syntax using styled-components
```

Comment:
```suggestion
  // Example of styling syntax using emotion
```
There's also a mention of styled-components (& link to https://styled-components.com/) above on line 107 - let's update this to reflect that we're now using emotion.

Reply:
Cool, will create a task to rewrite all this documentation once we merge the Chakra PR. Should be kind of aligned with what we are writing in the migration guide.